modules/oxide/src/less/theme/components/comments/comment.less

```diff
@@ -51,37 +51,39 @@
     font-weight: 400;
     padding: 8px 0 8px 0;
     color: @comment-text-color;
-  
+
     line-height: 28px;
   }
 
   .tox-comment {
     background: @comment-background-color;
     border: @comment-border;
     border-radius: @comment-border-radius;
-    box-shadow: @comment-box-shadow;
     padding: @comment-padding;
     position: relative;
+    &:hover {
+      box-shadow: @comment-box-shadow;
+    }
 
     &.tox-comment--selected {
       background-color: @comment-selected-background-color;
       border: 1px solid @comment-selected-border-color;
       box-shadow: 0px 4px 8px 0px rgba(34, 47, 62, 0.10);
-    
+
```

Comment:
Weird I wonder why there is space here?

Reply:
Space was there, and is now removed

Reply:
Yes I ment to say `was` good that you removed it but editorconfig should have removed it before.